crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,14 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def stop_chef_service
+    ssh_cmd("/usr/sbin/rcchef-client stop")
```

Comment:
That's a very good point; this should do an OS check and fail with an `unsupported OS` error if not SUSE.  Assuming we are on SUSE is not consistent with the rest of the Crowbar project.